docs: add "Evaluate Your Agent" page (#1306)
NVIDIA-NeMo/Gym: fern/versions/latest/pages/evaluate/evaluate-your-agent.mdx

---
title: "Evaluate Your Agent"
description: "Use NeMo Gym environments to score your own agent over HTTP."
position: 1
---

<Info>

> **Goal**: Learn how to evaluate your own Python agent using NeMo Gym environments — without rewriting it as a Gym server.

Reviewer: ```suggestion
**Goal**: Learn how to import and evaluate your own agent in NeMo Gym with minimal changes.
```
